requirements-cpu.txt

```diff
@@ -0,0 +1,4 @@
+dm_haiku==0.0.12
+jax==0.4.25
+numpy==1.26.4
+sentencepiece==0.2.0
```

Comment:
to be honest I don't know why they don't put the requirements in the pyproject.toml so you can just go

```
pip install .[cpu]
```

But that's just me.

Reply:
obviously this is just an opinion and I'm not "with the project", but I would have put the dependencies in the pyproject.toml myself and make installation groups like "cpu".